README.md

```diff
@@ -35,10 +38,12 @@ For example:
 	}
 }
 ```
+The value `opentable` is referenced via `$(foo.bar)` in your response body.
+
 ```
-<foo><bar>opentable</bar></foo>
+<root><foo><bar type="string">opentable</bar></foo></root>
```

Comment:
If we wrap this with the <root></root> element, wouldn't have to specify the value via $(root.foo.bar.value)?